contracts/vesting/src/messages.rs

```diff
@@ -13,6 +13,27 @@ pub struct InitMsg {
 #[serde(rename_all = "snake_case")]
 pub struct MigrateMsg {}
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema, Default)]
+pub struct VestingSpecification {
+    start_time: Option<u64>,
+    period_seconds: Option<u32>,
+    num_periods: Option<u32>,
```

Comment:
If you're already passing `Option<VestingSpecification>` when creating vesting accounts, does it make sense for all the fields here to be wrapped in `Option` as well?

Reply:
Each field has a default, so any combination of them can be passed in,, hence the multitude of Options